Fanyak alt text add 4 (#19844)
mdn/content: files/en-us/web/css/border-radius/index.md

  <tbody>
    <tr>
      <td><em>radius</em></td>

>       <td><img alt="A light blue rectangle with a light gray border. All 4 corners have been traced with a thick black line." src="all-corner.png" /></td>

Reviewer: ```suggestion
      <td><img alt="A light blue rectangle with a light gray border. All 4 corners are rounded." src="all-corner.png" /></td>
```

    <tr>
      <td><em>top-left-and-bottom-right</em></td>
      <td>

>         <img alt="A light blue rectangle with a light gray border. The 2 corners on the top left and the bottom right have been traced with a thick black line." src="top-left-bottom-right.png" />

Reviewer: ```suggestion
        <img alt="A light blue rectangle with a light gray border. The 2 corners on the top left and the bottom right have are rounded." src="top-left-bottom-right.png" />
```

    <tr>
      <td><em>top-right-and-bottom-left</em></td>
      <td>

>         <img alt="A light blue rectangle with a light gray border. The 2 corners on the top right and the bottom left have been traced with a thick black line." src="top-right-bottom-left.png" />

Reviewer: ```suggestion
        <img alt="A light blue rectangle with a light gray border. The 2 corners on the top right and the bottom left are rounded." src="top-right-bottom-left.png" />
```

    </tr>
    <tr>
      <td><em>top-left</em></td>

>       <td><img alt="A light blue rectangle with a light gray border. The corner on the top left has been traced with a thick black line." src="top-left.png" /></td>

Reviewer: ```suggestion
      <td><img alt="A light blue rectangle with a light gray border. The corner on the top left is rounded." src="top-left.png" /></td>
```

    </tr>
    <tr>
      <td><em>top-right</em></td>

>       <td><img alt="A light blue rectangle with a light gray border. The corner on the top right has been traced with a thick black line." src="top-right.png" /></td>

Reviewer: ```suggestion
      <td><img alt="A light blue rectangle with a light gray border. The corner on the top right is rounded." src="top-right.png" /></td>
```

    </tr>
    <tr>
      <td><em>bottom-right</em></td>

>       <td><img alt="A light blue rectangle with a light gray border. The corner on the bottom right has been traced with a thick black line." src="bottom-right.png" /></td>

Reviewer: ```suggestion
      <td><img alt="A light blue rectangle with a light gray border. The bottom right corner is rounded." src="bottom-right.png" /></td>
```

    </tr>
    <tr>
      <td><em>bottom-left</em></td>

>       <td><img alt="A light blue rectangle with a light gray border. The corner on the bottom left has been traced with a thick black line." src="bottom-left.png" /></td>

Reviewer: ```suggestion
      <td><img alt="A light blue rectangle with a light gray border. The bottom left corner is rounded." src="bottom-left.png" /></td>
```
